react/components/Input/edge.css

```diff
@@ -1,3 +1,7 @@
 .hideDecorators::-ms-clear{
   display: none;
-}
\ No newline at end of file
+}
+
+.noAppearance {
```

Comment:
Is this designed only for IE Edge?

Reply:
I don't know what was the intention of the dev that added this property. I just want to remove the error on AMP pages as I've said in the PR description

Reply:
Agreed, but even though, if it does not belong to an Edge fix it should not appear within this file dedicated to css for Edge.

Reply:
you want me to move this to a `Input.css` file?

Reply:
Sounds good to keep the project organised. I also made another comment above.

Reply:
why we have a stylesheet specific to edge bugs? I think it makes more sense to rename this `edge.css` to `Input.css` and add a comment to the `hideDecorators` class. When I first saw this class I had no idea it was related to the Edge browser.

Reply:
This could be a solution indeed.